### ISSUE TITLE
Tertiary Buttons lack focus rectangle

### ISSUE THREAD

**Issue by mbgower** [carbon-design-system/carbon#3624]
## Environment
Themes and v10

> Operating system
Windows 7

> Browser
Chrome and Firefox

## Detailed description
The Tertiary Buttons do not have visible focus indicators in either v10 or Themes. Note that on the Themes pages, there is the 'Tertiary Buttons' section and then the "Tertiary Buttons (Field)". The latter HAS a focus rectangle on the Add button (but no others)

> What did you expect to happen?
I expected the buttons to receive a standard dotted rectangle focus indicator. 

> What happened instead?
There is no focus indicator.

> What WCAG 2.1 checkpoint does the issue violate?
[Focus Visible](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html)

## Steps to reproduce the issue

1. Open up the themes page and select Button from the left nav
1. Scroll down to the section labelled Tertiary Buttons
1. Click or otherwise put focus on a button
3. Continue to press Tab until you reach the Add (+) button of the 'Tertiary Button (field)'

## Additional information
I suspect this is because the focus indicator is a white line. For all other buttons, there is a fill, and so the white line shows up, but for the tertiary buttons, the fill for the button is white, and therefor the indicator is invisible.

**Comment by tw15egan** [comment]
Should have been fixed with https://github.com/carbon-design-system/carbon/pull/3598

https://deploy-preview-3598--the-carbon-components.netlify.com/component/button--tertiary.html

Let me know if these changes didn't address your issue and I'll reopen.
